src/services/authService.js

```diff
@@ -13,6 +13,8 @@ angular.module('openshiftCommonServices')
 //     sets the current user and token to use for authenticated requests
 //     if ttl is specified, it indicates how many seconds the user and token are valid
 //     triggers onUserChanged callbacks if the new user is different than the current user
+//   getUser()
+//     returns the user
```

Comment:
You should explain how this is different from `withUser` (`withUser` triggers a login if there is no user, whereas this method will return null or undefined.)

Reply:
Ok, added comments